content/concepts/transport.md

```diff
@@ -115,3 +115,190 @@ and some places in the codebase still use the "swarm" terminology.
 {{% /notice %}}
 
 [definition_switch]: /reference/glossary/#switch
+
+## QUIC
+
+Transport protocols continue to improve transport methods in an attempt to alleviate 
+the shortcomings of the transport layer. 
+
+{{% notice "note" %}}
+
+Recalling the purpose of transports
+
+The IP service model provides logical communication between hosts (or nodes) but 
+is considered a best-effort delivery service, as segment delivery is not guaranteed. 
+The primary responsibility of transport protocols is to extend the IP 
+delivery service between two end systems. TCP connects the unreliable service of IP 
+between end systems into a reliable transport service between processes (i.e., the 
+processes running on the end systems). The purpose of newer transport protocols is not 
+only to improve current transport methods but also to allow for efficient connections 
+that distributed and peer-to-peer network stacks can utilize, like libp2p.
+
+{{% /notice %}}
+
+We need a transport protocol that:
+
+- Understands streams 
+- Is not byte-ordered
+- Overcomes HOL blocking (Head-of-line blocking)
+- Overcomes the latency of connection setup
+- Overcomes the ossification risks of TCP
+
+and, ideally, keeps the guarantees of TCP.
+
+In 2014, a new transport protocol 
+called QUIC (which, at the time, stood for Quick UDP Internet Connections, but now 
+is only referred to as QUIC and does not use the original acronym) was launched as an 
+experiment on Google Chrome. It has since been refined and maintained by an official 
+working group under the IETF (Internet Engineering Task Force). 
+
+QUIC is a UDP-based multiplexed and secure transport. 
+The official standard is defined in [RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000).
+
+Being UDP-based, QUIC optimizes for speedy transmission, as opposed to the latency 
+that exists in HTTP leveraging TLS over TCP.
+
+A web browser connection typically entails the following (TCP+TLS+HTTP/2):
```

Comment:
There’s a nice graphics floating around in various QUIC presentations showing QUIC’s position in the OSI stack. Maybe it would be worth including that one here?

Reply:
Indeed, planning to add diagrams separately (across the concept docs being added) in separate PRs, but can add this one here as part of this (part of that is to also to avoid a bunch of migration with the assets with the new proposed structure)